discovery-provider/alembic/versions/6b5186e7d28f_replace_aggregate_plays_mat_view_with_.py

```diff
@@ -27,10 +27,33 @@ def upgrade():
                 SELECT FROM information_schema.tables
                 WHERE table_name   = 'aggregate_plays'
             ) THEN
+                -- update indexing checkpoints based on current plays
+                WITH latest_play_id AS (
+                    SELECT MAX(id) AS id FROM plays
+                )
+                INSERT INTO indexing_checkpoints (tablename, last_checkpoint)
+                VALUES(
+                    'aggregate_plays',
+                    (COALESCE((SELECT id FROM latest_play_id), 0))
+                )
+                ON CONFLICT (tablename)
+                DO UPDATE SET last_checkpoint = EXCLUDED.last_checkpoint;
 
                 -- create aggregate_plays table
```

Comment:
worth maybe updating this comment to be

-- create aggregate_plays table from the latest checkpoint selected above